aplib.net-demo/Assets/Scripts/Inventory/KeyRing.cs

```diff
@@ -15,12 +16,19 @@ public class KeyRing : MonoBehaviour
     /// Checks the ID of all keys in the keyring against the inputted doorId, if a match is found, true is returned and the key is consumed.
     /// </summary>
     /// <param name="doorId">The ID of the inputted door that is then checked against all the keys in the keyring.</param>
-    /// <returns></returns>
-    public void KeyQuery(Door door)
+    /// <returns>True if the correct key is present otherwise False.</returns>
+    public bool KeyQuery(Door door)
```

Comment:
Are we gonna have something on the HUD that shows which keys you have collected? I don't think it needs to be  in this PR but it would be nice

Reply:
There is an issue for displaying keys in the HUD.